configs/openSUSE/opensuse.toml

```diff
@@ -0,0 +1,262 @@
+    ' perl-temp-file ',
+    ' hidden-file-or-dir .*/\.packlist',
+    ' hidden-file-or-dir .*/\.directory',
+    'perl-.*no-binary',
+    ' no-major-in-name ',
+
+# We check for that already
+    'dangling-relative-symlink',
+    ' lib-package-without-%mklibname',
+    ' requires-on-release',
+    ' non-executable-script /etc/profile.d/',
+    ' non-executable-script /var/adm/fillup-templates/',
+    ' init-script-name-with-dot ',
+    '.* statically-linked-binary /sbin/ldconfig',
+    '.* statically-linked-binary /sbin/init',
+    'valgrind.* statically-linked-binary',
+    'ldconfig-post.*/ddiwrapper/wine/',
+    'glibc\.\S+: \w: statically-linked-binary /usr/sbin/glibc_post_upgrade',
+    ' symlink-should-be-relative ',
+    ' binary-or-shlib-defines-rpath .*ORIGIN',
+    'libzypp.*shlib-policy-name-error.*libzypp',
+    'libtool.*shlib-policy.*',
+
+# Stuff that is currently too noisy, but might become relevant in the future
+    ' prereq-use',
+    ' file-not-utf8',
+    ' tag-not-utf8',
+    ' setup-not-quiet',
+    ' no-cleaning-of-buildroot ',
+    ' mixed-use-of-spaces-and-tabs ',
+    ' prereq-use ',
+
+# An issue with OBS, works with autobuild
+    ' no-packager-tag',
+    ' unversioned-explicit-provides ',
+    ' unversioned-explicit-obsoletes ',
+    ' service-default-enabled ',
+    ' non-standard-dir-perm ',
+    ' conffile-without-noreplace-flag ',
+    ' non-standard-executable-perm ',
+    ' jar-not-indexed ',
+    ' uncompressed-zip ',
+    ' %ifarch-applied-patch ',
+    ' read-error ',
+    ' init-script-without-chkconfig-postin ',
+    ' init-script-without-chkconfig-preun ',
+    ' postin-without-chkconfig ',
+    ' preun-without-chkconfig ',
+    ' no-dependency-on locales',
+    ' incoherent-version-in-name',
+    ' binary-or-shlib-defines-rpath',
+    ' executable-marked-as-config-file',
+    ' log-files-without-logrotate',
+    ' hardcoded-prefix-tag',
+    ' no-documentation',
+    ' multiple-specfiles',
+    ' no-default-runlevel ',
+    ' setgid-binary ',
+    ' non-readable ',
+    ' postin-without-ghost-file-creation ',
+
+# Exceptions for filelist checks
+    'nfs-client\.\S+: \w: filelist-forbidden-backup-file /var/lib/nfs/sm.bak ',
+    'perl\.\S+: \w: filelist-forbidden-perl-dir ',
+    'info\.\S+: \w: info-dir-file .*/usr/share/info/dir',
+
+# These packages are used for CD creation and are not supposed to be
+# installed. It's still a dirty hack to make an exception. The
+# packages should either be built in a separate project with
+# different config or file be put somewhere below /opt/suse/*
+    '(?:dosutils|skelcd|installation-images|yast2-slide-show|instlux|skelcd-.*|patterns-.*)\.\S+: \w: filelist-forbidden-fhs23 /CD1',
+
+# Too noisy, and usually not something downstream packagers can fix
+    ' incorrect-fsf-address ',
+    ' no-manual-page-for-binary ',
+    ' static-library-without-debuginfo /usr/lib(?:64)?/ghc-[\d\.]+/',
+
+# The libre mess
+    'libre(?:ssl|office|cad)[^\:]+: \w: shlib-policy-',
```

Comment:
This should be sorted in new rpmlint i think

Reply:
Do you mean all `Filter` values? What to do with the grouping comments?

Reply:
I mean shlib-policy is no longer scanning anything like this so it should not throw the error.